PlaceRouteHierFlow/router/A_star.cpp

```diff
@@ -131,6 +134,13 @@ int A_star::Manhattan_distan(int sindex, Grid &grid) {
   int dis = *it;
```

Comment:
The original code was doing it wrong looks like. `*(Mdis.end())` is invalid.

Reply:
yes you are right, thanks